about.html

```diff
@@ -7,67 +7,100 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
+
+   
+    <div class="navigation">
+    
+         <img src="img/lambda-black.png" alt="strategy"> 
+    
+    
+            <nav>
+              <a href="http://127.0.0.1:5500/Sprint-Challenge--User-Interface/#">Home</a>
```

Comment:
Great job making with styling, just a visual ease of readability thing, try using 1 blank line between adjacent `divs` or elements and just use tabbing to indicate nesting instead of leaving big blank spaces like this.